did-utils/src/crypto/x25519.rs

```diff
@@ -30,12 +44,33 @@ impl KeyMaterial for X25519KeyPair {
     }
 }
 
-
 impl Generate for X25519KeyPair {
+    /// Generates a new X25519 key pair.
+    ///
+    /// If the initial seed is empty or invalid, a random seed will be generated.
```

Comment:
What I mean here is that there is no initial seed argument.
```suggestion
    /// A random seed will be generated.
```

Reply:
See https://github.com/adorsys/didcomm-mediator-rs/pull/133#discussion_r1696714222